<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="KADITPC062024-1" failures="1" tests="1" name="LoginTest" time="2.043" errors="0" timestamp="2024-09-16T12:21:28 IST">
  <testcase classname="Intro.LoginTest" name="testLogin" time="0.0">
    <failure type="java.lang.RuntimeException" message="java.io.FileNotFoundException: Login_test_data.csv (The system cannot find the file specified)">
      <![CDATA[java.lang.RuntimeException: java.io.FileNotFoundException: Login_test_data.csv (The system cannot find the file specified)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: java.io.FileNotFoundException: Login_test_data.csv (The system cannot find the file specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:213)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:152)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:106)
at java.base/java.io.FileReader.<init>(FileReader.java:60)
at fileutility.CSVUtils.readCSV(CSVUtils.java:14)
at fileutility.Dataproviders.loginDataprovider(Dataproviders.java:15)
... 25 more
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
</testsuite> <!-- LoginTest -->
